plasmapy/physics/parameters.py

```diff
@@ -1191,22 +1192,27 @@ def inertial_length(n: u.m**-3, particle='e-'):
     Warns
     -----
     ~astropy.units.UnitsWarning
-        If units are not provided, SI units are assumed
+        If units are not provided and SI units are assumed.
 
     Notes
     -----
-    The particle inertial length is also known as an particle skin depth and is
-    given by:
+    The inertial length of a particle of species :math:`s` is given by
```

Comment:
Likewise here, the `:math:` part doesn't add much to a single letter besides making it the single most beautiful *s* to ever get sent to Mathjax from a browser. I love it for complex expressions, though.